packages/ra-supabase-ui-materialui/src/ForgotPasswordPage.tsx

```diff
@@ -0,0 +1,35 @@
+import * as React from 'react';
+import { ReactNode } from 'react';
+import { AuthLayout } from './AuthLayout';
+import { ForgotPasswordForm } from './ForgotPasswordForm';
+
+/**
+ * A component that renders a page for setting the current user password through Supabase.
+ * Can be used for the first login after a user has been invited or to reset the password.
```

Comment:
Probably a wrong copy/paste, as this applies to `SetPasswordPage` but not this page IMO